### PR TITLE
[BUG] Fix tests in TestSpatialSoftArgmax2d

### PR DESCRIPTION
**Pull request by anibali**
Fixes https://github.com/kornia/kornia/issues/535.

* The `normalized_coordinates` parameter was being used incorrectly.
* A higher input element value is required to get "crisp" coordinate results, since `spatial_soft_argmax2d` softmaxes the input (not to be confused with `spatial_softargmax_2d`, which does not).

### CODE REVIEW

File: test/geometry/test_spatial_softargmax.py [hunk 1]
```diff
@@ -74,57 +74,54 @@ def test_smoke_batch(self, device):
         m = kornia.SpatialSoftArgmax2d()
         assert m(input).shape == (2, 1, 2)
 
-    @pytest.mark.skip()
-    def test_top_left(self, device):
+    def test_top_left_normalized(self, device):
         input = torch.zeros(1, 1, 2, 3).to(device)
-        input[..., 0, 0] = 10.
+        input[..., 0, 0] = 1e16
```
Comment (edgarriba) [inline test/geometry/test_spatial_softargmax.py:79]: just curious, does it have to be that huge ?

Comment (anibali) [reply test/geometry/test_spatial_softargmax.py:79]: Probably not, no. That's just the first value that I tried.
